tests/unit/binlog_test.cpp

```diff
@@ -0,0 +1,132 @@
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE BINLOG library test
+
+#include <boost/test/included/unit_test.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "eblob_wrapper.h"
+
+#include <library/datasort.h>
```

Comment:
It is a local include, so it should be in `""`

Reply:
done